app/backend/approaches/chatreadretrieveread.py

```diff
@@ -99,13 +82,19 @@ async def run_until_final_call(
         if not isinstance(original_user_query, str):
             raise ValueError("The most recent message content must be a string.")
 
+        reasoning_model_support = self.GPT_REASONING_MODELS.get(self.chatgpt_model)
```

Comment:
@pamelafox  need to discuss what's the behavior we want when you use a reasoning model that doesn't stream?

Reply:
Option: Pass a "don't stream" config down to the frontend to avoid this issue

Reply:
Option: error.py edit for a more friendly error